src/index.js

```diff
@@ -58,7 +58,8 @@ const processor = postcss.plugin('postcss-modules-scope', function(options) {
     const generateScopedName =
       (options && options.generateScopedName) || processor.generateScopedName;
 
-    const exports = {};
+    let exports = Object.create(null);
+    // let exports = {};
```

Comment:
Remove comment, and why don't use `const`?

Reply:
done!